netmap/netmap_contract.go

```diff
@@ -20,10 +20,13 @@ type (
 
 	netmapNode struct {
 		node  storageNode
-		state nodeState
+		state NodeState
 	}
 
-	nodeState int
+	// NodeState is an enumeration for node states.
+	// NOTE: For historical reasons numerical values differs
```

Comment:
```suggestion
	// NOTE: For historical reasons numerical values differ
```

Reply:
Actually, it is my mistake, looks like they are the same, just that we had some logic in IR node.